### PR TITLE
Make flex-grow change so that we can avoid a Who's on First discussion.

### PR DESCRIPTION
**Pull request by jimkang**

### CODE REVIEW

**Comment by jimkang** [comment]
Looks like this.
Screenshot_2021-03-12 Rehydrate

**Comment by dpmcnevin** [comment]
I wonder if it's it's some difference in mac vs. linux firefox somehow?

`flex-grow: 1`

image

`flex: 1`

image

File: app.css [hunk 1]
```diff
@@ -127,8 +127,7 @@ ul {
 
 .pane.index-pane .index-media .index-media-content {
   /* Take the remaining width */
-  /* flex: flex-grow; */
-  flex: 1;
+  flex-grow: 1;
```
Comment (dpmcnevin) [inline app.css:130]: Changing to 

```css
  flex-grow: 1;
  flex-basis: 0%;
```

gave the look i was after with the bigger image

**Comment by dpmcnevin** [comment]
Closing this since I added the changes to the other PR

**Pull request closed without merging**